zerver/decorator.py

```diff
@@ -264,7 +264,10 @@ def _wrapped_func_arguments(request, api_key=REQ(),
                 return view_func(request, user_profile, *args, **kwargs)
             except Exception:
                 if request.content_type == 'application/json':
-                    request_body = ujson.dumps(ujson.loads(request.body), indent=4)
+                    try:
+                        request_body = ujson.dumps(ujson.loads(request.body), indent=4)
+                    except ValueError:
+                        request_body = str(request.body)
```

Comment:
I think if this does throw an exception, we should probably add something to the message below...